src/java/picard/sam/CreateSequenceDictionary.java

```diff
@@ -53,12 +53,25 @@
  * SAMRecords, and the header contains only sequence records.
  */
 @CommandLineProgramProperties(
-        usage = "Read fasta or fasta.gz containing reference sequences, and write as a SAM or BAM file with only sequence dictionary.\n",
-        usageShort = "Creates a SAM or BAM file from reference sequence in fasta format",
+        usage = CreateSequenceDictionary.USAGE_SUMMARY + CreateSequenceDictionary.USAGE_DETAILS,
+        usageShort = CreateSequenceDictionary.USAGE_SUMMARY,
         programGroup = Fasta.class
 )
 public class CreateSequenceDictionary extends CommandLineProgram {
-
+    static final String USAGE_SUMMARY = "Creates a SAM or BAM file from reference sequence in fasta format.  ";
+    static final String USAGE_DETAILS = "Writes a SAM/BAM file from a fasta-formated reference sequence.  " +
+            "The output SAM file contains a header, but no SAMRecords and the header contains only sequence records." +
+            "<br /><br />" +
+            "The tool reads both fasta- or fasta.gz-containing reference sequences."  +
+            "" +
+            "<h4>Usage example:</h4>" +
+            "<pre>" +
+            "java -jar picard.jar CreateSequenceDictionary \\ <br />" +
+            "      R=referenceSequence.fasta \\ <br />" +
+            "      O=refSeq.dict" +
```

Comment:
Based on this I suspect the above docstring is wrong or at the very least badly misstated ; this tool doesn't produce a bam file , it produces a sequence dictionary (.dict file), which I guess you could say *looks* like a SAM file without SAMrecords -- but that's only because the part of the header that has sequence records *is a sequence dictionary*. Whoever wrote that original docstring had a very convoluted way of describing things.